TAInformation/Models/user.py

```diff
@@ -2,14 +2,29 @@
 
 
 class User(ABC):
+
+    # precondition: id: Int, name: String, email: String, home_address: String, role: String, phone: String
+    # post condition: creates a user and sets fields
+    def __init__(self, id, name, password, email, home_address, role, phone):
```

Comment:
You can use type hints so that every time a User-derived class is initialized, the hints for what type should be used for each parameter show up.  For example:

def __init__(self, id: int, name: str, password: str, ...) [I typed double underscores at both ends of init, but apparently Github won't display them, because they use underscore as formatters.]

Also, I'm pretty sure we were planning on using a enum or int to represent roles.  I will send a message on Discord for clarification, maybe I remember talking about this about a week ago.